completions/bash/crio

```diff
@@ -19,6 +19,7 @@ h
 --address
 --allowed-devices
 --apparmor-profile
+--automatically-reload-registries
```

Comment:
I know that @haircommander asked you to rename this before, but we could keep this shorter and also aligned with the other registries-related options we already have, per:

_(an excerpt from the man page)_

```
        '--registries-conf'
        '--registries-conf-dir'
```

So adding `---registries-watch` or `--registries-watcher` would be short and to the point. The `--automatically-reload-registries` is really a bit of a mouthful. Thoughts?

Reply:
What do you think about `--auto-reload-registries`? I believe clarity in variable names, reflecting their functionality, is important.

Reply:
@sohankunkerkar, that looks nice! Thank you!